packages/libicu-deb/libicu-deb.pacscript

```diff
@@ -0,0 +1,9 @@
+name="libicu-deb"
+version="72.1"
+gives="libicu${version:0:2}"
+url="http://archive.ubuntu.com/ubuntu/pool/main/i/icu/libicu72_${version}-3ubuntu2_amd64.deb"
```

Comment:
You should probably add the string splicing to this as well

Reply:
Yeah but pacup fails to parse it :/
So anyway I would have to change it manually

Reply:
Oh true..